### ISSUE TITLE
Remove references to "please" from CH 31

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#65342]
## Issue Description
_The CH 31 form for current references "Please enter..." as part of input field validation, but "please" was depreciated in 2022 in favor of direct language._
---
## Tasks
- [ ] Remove all references to "Please..." For example, "Please enter your name." would become "Enter your name.".

## Acceptance Criteria
- [ ] All depreciated instances of "please" have been removed from the form.
---

**Comment by steele-lm** [comment]
Needs to clarify with CAIA if removing "please" only applies to certain cases and not other (e.g., instructions).

**Comment by steele-lm** [comment]
CAIA clarified that all instances of please should be removed where possible.

**Comment by zack** [comment]
@steele-lm This, along with the other two tickets we have for work to remove "please" can only be partially done on a form-by-form basis. Forms pull in generic error messaging that is used across _all_ forms that has the word "please" in it. It's impossible to remove for only some of the forms (well, it's not impossible, but it would require writing *more* code). Do we have sign-off from Platform to just go ahead and also remove it form the generic, shared error messages that are used across _all_ forms?

**Comment by steele-lm** [comment]
@zack Yes, proceed to remove the instances of "please" that we manage. Shared error messaging will have to be addressed at the Platform level.
